docs/user-guide/irf/index.rst

```diff
@@ -30,11 +48,24 @@ where:
 * :math:`\Phi(p_{\rm true}, E_{\rm true})` is the sky flux model  (unit: :math:`{\rm m}^{-2}\,{\rm s}^{-1}\,{\rm TeV}^{-1}\,{\rm sr}^{-1}`)
 * :math:`t_{\rm obs}` is the observation time:  (unit: :math:`{\rm s}`)
 
+Residual instrumental background rate
+-------------------------------------
+
+The response :math:`{\rm Bkg}(p, E)` is coming from atmospheric cosmic-rays that are mis-classified as gamma-ray candidates.
+They originate from hadrons (mainly protons) and leptons (mainly electrons), depending on the energy. These events
+constitute an irreducible source of background when studying gamma-ray emissions; they are also subjects of analysis
+(e.g. cosmic ray spectrum, charge composition, isotropy and dipole search), which is beyond the scope of this documentation.
+
+:math:`{\rm Bkg}(p, E)` predicts the rate of such events. This response is complex to derive by the observatories, in order to
+cover the whole observational phase space (p, E) and the instrument variations (atmosphere and detectors). Most of the
+time, real events are used to build such instrument response. This rate is then delivered by the observatories.
+In the DL3 format, this IRF is distributed for each observing run for the :term:`IACT` observatories.
```

Comment:
Maybe also mention that in the absence of a supplied background irf, one can use a measurement of the off counts in the FoV, and refer to eg, the reflected regions page